src/NodeServices/IAnalyticsClient.cs

```diff
@@ -559,7 +560,17 @@ public interface IAnalyticsClient
         /// <param name="description">Event description</param>
         /// <param name="value">A metric value associated with the event</param>
         /// <returns>Event as IDisposable</returns>
-        Task<IDisposable> CreateTaskCommandEvent(string name, string description, int? value);
```

Comment:
this was introduced 3 weeeks ago (for 3.0).
I think breaking the interface should still be ok...